xbmc/network/WakeOnAccess.cpp

```diff
@@ -194,7 +188,11 @@ class ProgressDialogHelper
 
     if (m_dialog)
     {
-      m_dialog->SetLine(1, CVariant{line1});
+      m_dialog->SetLine(0, CVariant{line1});
+
+      int nest_level = NestDetect::Level();
+      if (nest_level > 1)
+        m_dialog->SetLine(1, CVariant{ StringUtils::Format("Nesting:%d", nest_level) });
```

Comment:
Sure. If it's rarely used and only meant for debugging i see no reason to keep a dialog and add another string that has to be translated.